app/_kong_plugins/ai-mcp-oauth2/examples/configure-the-ai-mcp-oaut2-plugin.yaml

```diff
@@ -0,0 +1,44 @@
+description: 'Configure the AI MCP Oauth2 plugin'
```

Comment:
let's mention this plugin in "Technical Preview" state. If we need more explaination we can mention the OAuth2 spec in MCP is new and evolving, while majority of IdP has not fully implemented what the spec asks, this plugin is subject to change.

Reply:
@tomek-labuk 
Sure. We have a config `metadata_endpoint` in this plugin to configure the endpoint.